### PR TITLE
Update clone instructions for the public repo

### PR DESCRIPTION
**Pull request by embeddery**
## Description

Per title.

## Checklist
- [x] Investigated and inspected CI test results

## Testing Performed

n/a just some docs.

### CODE REVIEW

File: README.md
Comment (ehashman) [inline README.md:114]: Suggest `mkdir -p` as this will fail if the directories already exist

**Review by ehashman — APPROVED** [review]
/lgtm

**Comment by embeddery** [comment]
Can a maintainer run CI plz?

**Comment by gavin-stackrox** [comment]
Cannot merge without CI :(

**Pull request closed without merging**
